js/globals.ts

```diff
@@ -104,3 +104,9 @@ export type TextDecoder = textEncoding.TextDecoder;
 window.performance = new performanceUtil.Performance();
 
 window.workerMain = workers.workerMain;
+
+// @ts-ignore
```

Comment:
Had to put it there otherwise, Rollup bails out on TS6196

Reply:
I really don't like using this... I would just export the interface then the error will go away.

Reply:
Also it might make sense to only deal with exported interfaces from this module.